Provider/src/main/java/com/spotify/confidence/client/ConfidenceRemoteClient.kt

```diff
@@ -20,6 +20,8 @@ import java.net.HttpURLConnection
 
 class ConfidenceRemoteClient : ConfidenceClient {
     private val clientSecret: String
+    private val sdkId: String
```

Comment:
if we add these 2 to another object (creating a new type), then we can add more stuff to it in the future if needed easier.